packages/next/build/swc/index.js

```diff
@@ -105,6 +105,13 @@ function loadNative() {
     } catch (e) {}
   }
 
+  for (const triple of triples) {
+    try {
+      bindings = require(`next/native/next-swc.${triple.platformArchABI}.node`)
+      break
+    } catch (e) {}
+  }
+
   if (!bindings) {
```

Comment:
Do we need this block any more?

Reply:
I don't think we should for most cases although we could leave it in case users want to manually install the package and skip our custom install script 🤔